api/middleware.go

```diff
@@ -185,7 +185,7 @@ func (a *API) limitEmailSentHandler() middlewareHandler {
 					return c, nil
 				}
 
-				if err := tollbooth.LimitByRequest(lmt, w, req); err != nil {
+				if err := tollbooth.LimitByKeys(lmt, []string{"email_functions"}); err != nil {
```

Comment:
damnn i didn't know you could just do it like this to enforce the rate limit across all the endpoints 👀